File: allSpice/Models/Recipe.cs
namespace allSpice.Models
{
    public class Recipe
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Instructions { get; set; }
        public string Img { get; set; }
        public string Category { get; set; }
        public string CreatorId { get; set; }

        public Profile Creator { get; set; }
    }

    public class FavoriteRecipe : Recipe
    {
        public int FavoriteId { get; set; }
    }
}

File: allSpice.client/src/models/Recipe.js
export class Recipe {
    constructor(data) {
        this.id = data.id
        this.favoriteId = data.favoriteId || 0
        this.title = data.title
        this.instructions = data.instructions
        this.img = data.img
        this.category = data.category
        this.creatorId = data.creatorId
        this.creator = data.creator
    }
}